nengo_bones/templates/docs/conf.py.template

```diff
@@ -61,7 +61,7 @@ nbsphinx_timeout = -1
 
 # -- notfound.extension
 notfound_template = "404.html"
-notfound_urls_prefix = "{{ notfound_urls_prefix }}"
+notfound_urls_prefix = "/{{ pkg_name | replace("_", "-") }}/"
```

Comment:
Safer to use `repo_name` and split it on `/` as this will fail for, e.g., when we do `nengo/nengo-core` (which will still have `nengo` as the `pkg_name`

Reply:
Wouldn't the documentation still be at `nengo.ai/nengo`?

Reply:
No, the location of the documentation is based on the repository's name, so if we move the repo to `nengo/nengo-core` (which I think is the plan) then it will move all the docs to `nengo.ai/nengo-core`. Which means that we should definitely put a bunch of redirects in the `nengo/nengo` metapackage before we actually do the move.